### DISCUSSION TITLE
Search query disappears if pressing arrow up

### DISCUSSION THREAD

**Discussion by spogulis** [community/community#106632] [Copilot Conversations]
### Select Topic Area

Bug

### Body

This is a serious flaw since a long query can be lost if the user accidentally presses Arrow Up on keyboard as this pastes the previously used query. Arrow down does not remedy this.

**Comment by gil-son** [comment]
Yeah, it's a problem!

Verify in setting that causes the Arrow Up key to paste the previous query. Explore the application settings to see if there's an option to disable this behavior
